pkg/skaffold/schema/latest/config.go

```diff
@@ -1419,6 +1419,9 @@ type KanikoArtifact struct {
 	// Defaults to the latest released version of `gcr.io/kaniko-project/executor`.
 	Image string `yaml:"image,omitempty"`
 
+	// ImagePullSecret is the name of the Kubernetes secret for pulling kaniko image and kaniko init image from a private registry
+	ImagePullSecret string `yaml:"imagePullSecret,omitempty"`
```

Comment:
Hello skaffold maintainers, is someone available to help? I was instructed to create  a separate pull request for the config change. It is only the 2 lines above. How are the unit tests / linter errors caused by the config change? How to resolve?